examples/pytorch/diffpool/README.md

```diff
@@ -16,8 +16,8 @@ How to run
 ----------
 
 ```bash
-python train.py --dataset ENZYMES --pool_ratio 0.10 --num_pool 1
-python train.py --dataset DD --pool_ratio 0.15 --num_pool 1  
+python train.py --dataset ENZYMES --pool_ratio 0.10 --num_pool 1 --epochs 1000
+python train.py --dataset DD --pool_ratio 0.15 --num_pool 1  --batch-size 10
 ```
 Performance
 -----------
```

Comment:
Please include a short description of the improvement of both memory and accuracy, and add statistics in README as supportive evidence.

Also, if applicable, please also include acc/memory from the original implementation.